package.json

```
@@ -24,25 +24,30 @@
   ],
   "files": [
     "index.js",
-    "regex.js"
+    "regex.js",
+    "*.d.ts"
   ],
-  "dependencies": {},
   "devDependencies": {
     "@unicode/unicode-13.0.0": "^1.0.0",
     "nyc": "^15.0.0",
     "prettier": "^2.0.0",
     "regenerate": "^1.0.0",
     "remark-cli": "^9.0.0",
     "remark-preset-wooorm": "^8.0.0",
+    "rimraf": "^3.0.0",
     "tape": "^5.0.0",
+    "typescript": "^4.0.0",
     "xo": "^0.36.0"
   },
   "scripts": {
     "generate": "node build",
     "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
     "test-api": "node test",
     "test-coverage": "nyc --reporter lcov tape test.js",
-    "test": "npm run generate && npm run format && npm run test-coverage"
+    "prebuild": "rimraf \"*.d.ts\"",
+    "build": "tsc",
```

```suggestion
    "build": "rimraf \"*.d.ts\" && tsc",
```

Not needed to split it out IMO

---

I totally agree! I only did this, because I used the same as in `unified`.